.buildkite/scripts/generate_libbeat_pipeline.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+source .buildkite/scripts/common.sh
+
+set -euo pipefail
+
+pipelineName="pipeline.libbeat-dynamic.yml"
+
+echo "Add the mandatory and extended tests without additional conditions into the pipeline"
+if are_conditions_met_mandatory_tests; then
+  cat > $pipelineName <<- YAML
+
+steps:
+
+  - group: "Mandatory Tests"
+    key: "mandatory-tests"
+    steps:
+      - label: ":linux: Ubuntu Unit Tests"
+        key: "mandatory-linux-unit-test"
+        command: ".buildkite/scripts/unit_tests.sh"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
```

Comment:
is `machineType` required in this particular case? Maybe4 it's a good idea to avoid relying on the provisioned but enforcing the required type of machine?

Reply:
@v1v, please take a look at the test results https://elastic.slack.com/archives/C0522G6FBNE/p1707152502952139?thread_ts=1707140315.980879&cid=C0522G6FBNE

Reply:
done